Makefile

```diff
@@ -18,3 +22,25 @@ buck-stop:
 
 buck-clean:
 	docker-compose -f cmd/buckd/docker-compose-dev.yml down -v --remove-orphans
+
+# local is what we run when testing locally.
+# This does breaking change detection against our local git repository.
+.PHONY: buf-local
+buf-local: $(BUF)
```

Comment:
This is now you use the Bingo Makefile vars. Here is make `buf-local` depend on `$(BUF)` which will make sure the correct version of `buf` is downloaded, built, and used.